dbt/macros/tests/test_generate_schema_name.sql

```diff
@@ -12,10 +12,6 @@
     {% endif %}
 {% endmacro %}
 
-{% macro mock_raise_compiler_error(_error) %}
-    {{ return("Compiler error raised") }}
-{% endmacro %}
```

Comment:
I realized in the course of developing this PR that it's trivial to adjust `mock_raise_compiler_error` to return the error string for the purposes of making sure that a specific error was raised, so I did this and moved the updated macro to `dbt/macros/tests/utils.sql` so that it could be used by the new `format_additional_select_columns` test.